src/common/placeholderImage.component.tsx

```diff
@@ -20,10 +17,11 @@ const PlaceholderImage = (props: PlaceholderImageProps) => {
         alignItems: 'center',
         justifyContent: 'center',
         textAlign: 'center',
-        color: theme.palette.text.primary,
+        color: (theme: Theme) => theme.palette.text.primary,
```

Comment:
```suggestion
        color: 'text.primary',
```

Reply:
I had to remove the useTheme as it wasn't work correctly